rich/console.py

```diff
@@ -2262,12 +2262,12 @@ def export_svg(
         clear: bool = True,
         code_format: str = CONSOLE_SVG_FORMAT,
         font_aspect_ratio: float = 0.61,
+        unique_id: Optional[str] = None,
```

Comment:
Hi - apologies in advance: maybe some unnecessarily verbose commentary.

I'm wondering whether an (replacement?  additional?) argument, `isolate: bool = True` could make sense here.

With `isolate=False`, the caller would be saying that they don't want/need unique identifiers to be included on the exported SVG.  That could match and perhaps improve upon the Airflow usecase (https://github.com/apache/airflow/pull/27960), producing smaller SVG artifacts from the [Breeze CI tooling](https://github.com/apache/airflow/blob/b5084bcef9c4b0304a1643f891e1b8c80c8b0621/BREEZE.rst), without requiring a [magic constant](https://github.com/apache/airflow/pull/27960/commits/4b56d3d7a56b34701ddcaeed921b55eab6244a5e#diff-c3cfdecea722cd31d4d30a978fb7d0ddd1c28538abee29fcfaf0040a5e2e3501R453).

As I understand it, the reason to generate per-tag identifiers was to limit the scope of the CSS rules emitted by the SVG export (necessary because [textually-embedded SVG elements in XML](https://www.w3.org/TR/SVG11/styling.html#Scope) share their stylesheets with the document).

[In SVG2, shadow-trees may provide a way to ensure that stylesheets are encapsulated](https://www.w3.org/TR/SVG2/struct.html#UseStyleInheritance) -- opening a migration path towards removal of the `unique_id` value (and maybe the `zlib` dependency).

So far, I'm unaware of any other codebases on GitHub that are passing custom values to the `unique_id` argument.

Reply:
Svg2 is a dead spec with (next to?) no browser support to my knowledge

Reply:
And zlib is part of the standard library

Reply:
With those responses noted, are there elements of the suggestion that you do agree with or do you consider it a no-go?